backend/api/Areas/Tools/Helpers/ImportPropertiesHelper.cs

```diff
@@ -193,7 +200,7 @@ private Entity.City GetOrCreateCity(string name)
                 else if (last.Length < 4) last = last.Substring(1, last.Length - 1);
 
                 // Check if the code is unique.
-                city = _cities.FirstOrDefault(c => c.Code == code);
+                city = _cities.FirstOrDefault(c => String.Compare(c.Code, code, true) == 0);
```

Comment:
Because we're not sending separate queries to the DB we have to handle case-sensitivity.